### PR TITLE
✨ 🦸 ✨  Hero update with Image component

### PR DESCRIPTION
**Pull request by LowLifeArcade**
### This PR: 
This addresses performance issues in #313 where we're getting a hit in performance from using old image formats. Hopefully the Image component fixes this hit.

Also added revalidation so that each day is a new hero image.

### Self-Review Checklist

- [x] PR title is clear with proper spelling and grammar
- [x] PR description contains a bulleted list of changes contained in the PR
- [x] PR links to relevant issues, with keywords to auto-close any issues fully resolved upon merge
- [x] All automated checks passed
- [x] Any variables introduced are named clearly and explicitly

### CODE REVIEW

**Comment by LowLifeArcade** [comment]
Looks like it worked!
Screen Shot 2021-11-03 at 4 12 42 PM

File: components/HeroSection.js [hunk 1]
```diff
@@ -54,18 +63,39 @@ function HeroSection({ main = false, image, inNeed = false }) {
...
+          z-index: 0;
+        }
+        .hero-image {
+          position: relative; !important
+        }
+        .description-box {
+          padding: 20px;
+          /* padding-bottom: 40px; */
+          background: linear-gradient(to right, rgba(0, 0, 0, 0) 5%, rgba(0, 0, 0, 0.473) 10% 70%, rgba(0, 0, 0, 0));
+        }
```
Comment (LowLifeArcade) [inline components/HeroSection.js:98]:
```suggestion
       
```
This should be removed

File: components/HeroSection.js [hunk 2]
```diff
@@ -12,7 +10,18 @@ function HeroSection({ main = false, image, inNeed = false }) {
...
+          <Image
+            alt="Background"
+            className="hero-image"
+            layout="fill"
+            objectFit="cover"
+            src={image}
+          />
+        </div>
+      )}
+      <div className={main && 'hero'}>
```
Comment (Sahil-101) [inline components/HeroSection.js:24]: This looks well, it's good to have an image rather than a background image which we had.

**Review by Sahil-101 — COMMENTED** [review]
All good! we got a pretty boost in performance 😄

**Pull request merged**